src/components/App.tsx

```diff
@@ -3,8 +3,11 @@ import Landing from 'ribbit/components/Landing';
 import { useAuthStore } from 'ribbit/state/auth';
 import React from 'react';
 import supabase from 'ribbit/supabase';
-import Header from 'ribbit/components/Header';
+import HeaderAuth from 'ribbit/components/HeaderAuth';
 import PageNotFound from 'ribbit/components/PageNotFound';
+import { AppShell, Burger, MantineProvider, Title } from '@mantine/core';
+import { useDisclosure } from '@mantine/hooks';
+import styles from 'ribbit/components/App.module.css';
```

Comment:
I dunno if you used [CSS Modules](https://github.com/css-modules/css-modules) in your lessons, I don't have a major preference for or against them but they're convenient with Vite and Mantine recommends using them with their components.